powershell/upload/README.md

```diff
@@ -0,0 +1,142 @@
+# File API PowerShell upload examples
+
+A collection of examples to upload files with the File API using **PowerShell**.
+
+## Prerequisites
+
+- The script was prepared for PowerShell version 5.1 or above. With lower versions it might not work properly.
+- Files to be uploaded cannot be bigger than 100 megabytes.
+
+## Getting Started
+
+Upload the **file-api-integration-examples** repository.
```

Comment:
The replace tool trolled you here.
It's 'Download the [...]'.

Reply:
I will change it